projects/aws/eks-anywhere-build-tooling/Makefile

```diff
@@ -16,7 +16,7 @@ BUILD_TARGETS=local-images
 RELEASE_TARGETS=images
 
 FETCH_BINARIES_TARGETS=eksa/fluxcd/flux2 eksa/kubernetes-sigs/cluster-api \
-	eksa/kubernetes-sigs/kind eksa/replicatedhq/troubleshoot eksa/vmware/govmomi eksd/kubernetes/client eksa/helm/helm eksa/tinkerbell/tink \
+	eksa/kubernetes-sigs/kind eksa/replicatedhq/troubleshoot eksa/vmware/govmomi eksd/kubernetes/client eksa/helm/helm eksa/tinkerbell/tink/2b1eddcdf90082499b7663ca3c47e77dc12ad34c \
```

Comment:
I think the path also has the build number in it, as `<build_number>-<commit_hash>`.